pkg/store/project/project.go

```diff
@@ -69,8 +69,7 @@ func (s *store) Create(db *gorm.DB, project *model.Project) error {
 	return db.Create(&project).Error
 }
 
-type GetListProjectInput struct {
-	Status string `json:"status"`
-	Name   string `json:"name"`
-	Type   string `json:"type"`
+// Exist return true/false if project exist or not
+func (s *store) Exist(db *gorm.DB, id string) bool {
```

Comment:
If you check it by comparing `error` with `nil`, we possibly missing some errors caused by DBMS